src/js/components/search/SearchResults.jsx

```diff
@@ -18,6 +18,8 @@ const propTypes = {
     isMobile: PropTypes.bool,
     filterCount: PropTypes.number,
     showMobileFilters: PropTypes.bool,
+    requestsComplete: PropTypes.bool,
+    noFiltersApplied: PropTypes.bool,
     toggleMobileFilters: PropTypes.func,
     clearAllFilters: PropTypes.func
```

Comment:
Mobile "Clear all filters" button is not working because this prop is no longer passed down from `SearchPage`. Can that button be removed since we have the reset button?

Reply:
Removed